src/bootstrap-table.js

```diff
@@ -830,7 +830,9 @@
                     that.options.singleSelect = true;
                 }
 
+                html.push('<p>');
```

Comment:
This is for a custom implementation of the bootstrap table not related to the auto refresh extension. I created the pull request for auto refresh 7 months ago and it hasn't been accepted.